dashboard/app.yaml

```diff
@@ -33,7 +33,11 @@ resources:
 env_variables:
   REDISHOST: '10.25.27.107'
   REDISPORT: '6379'
-  TEST_NAME_PREFIXES: 'pt-nightly,pt-2.0,tf.2.14.0,tf.nightly,tf.se.nightly,tf.exp,%-1vm,jax,flax,pax-stable,pax-nightly,mp-jax,mp-pax,mp-pt'
+<<<<<<< Updated upstream
```

Comment:
I think this is a the leftover of a merge, needs to be cleaned up.

Reply:
Fixed